Dockerfile

```
@@ -1,4 +1,4 @@
-FROM centos/ruby-25-centos7
+FROM quay.io/centos7/ruby-25-centos7
```

Should we just go straight to the official image ?

```suggestion
FROM registry.access.redhat.com/ubi7/ruby-27
```

---

So I tried and we can't have the required packages with UBI8 without subscriptions:

- rh-mysql80
- imagemagick 

We could use the community ones but that means there is not much value using UBI8, lets stick to CentOS 7 for now